package.json

```diff
@@ -4,6 +4,22 @@
   "private": true,
   "license": "apache-2.0",
   "homepage": "https://realm.io",
+  "scripts": {
+    "build": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "./packages/realm:build",
+        "./packages/babel-plugin:build",
+        "./packages/mocha-reporter:build",
+        "./packages/realm-react:build",
+        "./packages/realm-tools:build",
+        "./packages/realm-web:build",
+        "./integration-tests/tests:build"
+      ]
+    }
```

Comment:
@takameyer this is the build script you requested. It only mentions the "leaf" packages as the other packages are transitively built.